interface/main/calendar/modules/PostCalendar/pntemplates/default/views/week/ajax_template.html

```diff
@@ -421,7 +422,7 @@
         if ($Date == $dateFmt) { $currclass= "week_currday"; }
         echo "<td align='center' class='week_dateheader $currclass' style='width:".$colWidth."%;' >";
         echo "<a href='$gotoURL'>";
-        echo xl(date("D", strtotime($date))) . " " . date("m/d", strtotime($date));
+        echo xl(date("D", strtotime($date))) . " " . oeFormatShortDate($date);
```

Comment:
could add a second a parameter to this function called $options or something like that. And 1 option would be 'no_year'. This shouldn't be too tough to then incorporate into the following function(which is a small straightforward function):
https://github.com/matrix-israel/openemr/blob/690852ed75f7408919a1c887c5efca44d0b713dc/library/formatting.inc.php#L20-L30